docs/index.rst

```diff
@@ -33,5 +33,6 @@ Contents
    api
    websockets
    contribute
+   database
```

Comment:
I guess it makes sense to have this as documentation for developers instead if a section just talking about the databases